conda_build/metadata.py

```diff
@@ -1316,9 +1317,11 @@ def parse_until_resolved(
     @classmethod
     def fromstring(cls, metadata, config=None, variant=None):
         m = super().__new__(cls)
-        if not config:
-            config = Config()
-        m.meta = parse(metadata, config=config, path="", variant=variant)
+        m.path = ""
+        m._meta_path = ""
+        m.requirements_path = ""
```

Comment:
initialization missing that was discovered by the newly added unit test

Reply:
Huh!